app/controllers/api/v1/bulk_export_controller.rb

```diff
@@ -17,60 +17,55 @@ def index
         else
           exports = @api_user.bulk_exports
         end
-        render json: exports.to_json(except: [:updated_at, :user_id, :collection_id], include: {:collection => {only: [:title, :slug]}})
+        render json: exports.to_json(except: [:updated_at, :user_id, :collection_id], include: { collection: { only: [:title, :slug] } })
       else
         render status: 401, json: 'You must use an API token to access bulk exports'
       end
     end
 
-
     def start
       if @api_user
         collection_slug = params[:collection_slug]
-        if collection_slug
-          collection = Collection.where(slug: collection_slug).first
-          if collection.nil?
-            render status: 404, json: "No collection exists with the slug #{collection_slug}"
-            return
-          end
+        collection = Collection.where(slug: collection_slug).first
```

Comment:
We only have :set_api_user in the filter. We also expect :collection_slug instead of :collection_id so it won't be set using our before filters.

I think what we have now is ideal. We want to throw the message `          render status: 404, json: "No collection exists with the slug #{collection_slug}"
`  since this is an API endpoint rather than a webpage. This allows user to know the error in their api calls.

Reply:
You've convinced me.